package.xml

```diff
@@ -8,7 +8,7 @@
   <author email="ayrton04@gmail.com">Tom Moore</author> 
   <maintainer email="ayrton04@gmail.com">Tom Moore</maintainer>
 
-  <license>BSD</license>
+  <license>Apache License 2.0</license>
```

Comment:
I'm not a legal expert, but I'd like to know what the ramifications are of changing this license, if any.